Formula/profanity.rb

```diff
@@ -29,13 +31,19 @@ class Profanity < Formula
   depends_on "libotr" => :recommended
   depends_on "gpgme" => :recommended
   depends_on "terminal-notifier" => :optional
+  # this always uses homebrewed python version
```

Comment:
OK, I just learned that I can use `depends_on :python ...` which automatically selects the python version.
That solves my problem, I think.